packages/components/src/components/tabs-bar/tabs-bar.tsx

```diff
@@ -190,6 +198,7 @@ export class TabsBar {
 
   private setTabElements = (): void => {
     this.tabElements = getHTMLElements(this.host, 'a,button');
```

Comment:
I think validation should happen here, otherwise later added children are not validated anymore (see `onSlotchange`).
Perhaps it would be simpler to create a new function similar to `getOnlyChildOfKindHTMLElementOrThrow()` which calls `getHTMLElements()`, does the validation and returns the results if validation passes.

Reply:
Yes, I also thought about the missing validation on `slotChange`. Then I decided to only validate on `disconnectedCallback()` otherwise the error was always exposed twice in the log because `slotChange` does also fire on first load?

Reply:
refactored error handling